dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
We really just need a class for IRIs that are named, but otherwise follow the same conventions/use cases as RDF blank nodes (ie describe reification, multicomponent structures, etc) and are not guaranteed to persist across releases.

We could create a class in the monarch data Monarch:Pseudonymous and use this to label nodes in SciGraph (as either anonymous, or pseudonymous)?